test/Concurrency/issue-70019.swift

```diff
@@ -0,0 +1,17 @@
+// RUN: %target-typecheck-verify-swift -swift-version 5
+// RUN: %target-typecheck-verify-swift -swift-version 5 -enable-library-evolution -I %t
+
+// REQUIRES: concurrency
+
+// https://github.com/apple/swift/issues/70019
+
+@usableFromInline
+struct Foo { // expected-note {{consider making struct 'Foo' conform to the 'Sendable' protocol}}
+  var integer: Int
+}
+
+struct Bar: Sendable {
+  // FIXME: This warning should only be thrown when it complies with library evolution mode
+  // FIXME: If @usableFromInline is removed from Foo the Sendable conformance is synthesized as expected.
```

Comment:
This seems like it is supposed to be a multiline FIXME, but it reads as 2 individual ones instead, with the second misleadingly describing *expected* behavior at that. I suggest keeping just the first marker.